yarn.lock

```diff
@@ -3201,36 +2850,43 @@
     "@types/node" "*"
     form-data "^4.0.0"
 
-"@types/node-forge@1.3.11":
+"@types/node-forge@1.3.11", "@types/node-forge@^1.3.0":
   version "1.3.11"
   resolved "https://registry.yarnpkg.com/@types/node-forge/-/node-forge-1.3.11.tgz#0972ea538ddb0f4d9c2fa0ec5db5724773a604da"
   integrity sha512-FQx220y22OKNTqaByeBGqHWYz4cl94tpcxeFdvBo3wjG6XPBuZ0BNgNZRV5J5TFmmcsJ4IzsLkmGRiQbnYsBEQ==
   dependencies:
     "@types/node" "*"
 
-"@types/node@*", "@types/node@22.15.2", "@types/node@^22.7.7":
+"@types/node@*", "@types/node@^22.7.7":
+  version "22.15.3"
+  resolved "https://registry.yarnpkg.com/@types/node/-/node-22.15.3.tgz#b7fb9396a8ec5b5dfb1345d8ac2502060e9af68b"
+  integrity sha512-lX7HFZeHf4QG/J7tBZqrCAXwz9J5RD56Y6MpP0eJkka8p+K0RY/yBTW7CYFJ4VGCclxqOLKmiGP5juQc6MKgcw==
+  dependencies:
+    undici-types "~6.21.0"
+
+"@types/node@22.15.2":
```

Comment:
I guess we could have also bumped this at the same time, but dependabot will probably get to it eventually?